index.js

```diff
@@ -14,7 +14,8 @@ function instance(system, id, config) {
 	self.command = null;
 
 	self.video_sources = [
-		{ id: 'SDI', label: 'SDI' },
+		{ id: 'SDI1', label: 'SDI 1' },
```

Comment:
You are missing the original "SDI" source

Reply:
Finally back at my computer long enough to take a look at this. Does it still need "SDI" in addition to "SDI 1" and "SDI 2" to work properly with the standard Teranex?

Reply:
yes, it needs. without that nothing happens.

Reply:
it needs something like this:

self.video_sources = [
		{ id: 'SDI', label: ‘SDI (Express)' },
		{ id: 'SDI1', label: ‘SDI1 (AV)' },
		{ id: 'SDI2', label: ‘SDI2 (AV)' },
		{ id: 'HDMI', label: 'HDMI' },
		{ id: 'Composite', label: 'Composite' },
		{ id: 'Component', label: 'Component' },
		{ id: 'Optical', label: 'Optical' }
	];

Reply:
Chris, do you want to fix this so I can pull this? There are several people who want to see this fix :)

Reply:
bump this thread. I hardly need this, please guys...

Reply:
@gabonacorp how can I help? Is it really just a matter of changing the `video_sources` array as you've requested?

Reply:
@josephdadams yes. if you can use my lines what I wrote on April 1 every Teranex devices gonna works properly. Teranex AV has two SDI input so it needs to be defined if it's SDI1 or SDI2. 

Every other commands are the same for booth versions of teranex.